content/docs/atom-guide/dev-guide/materials/_index.md

```diff
@@ -7,6 +7,8 @@ weight: 400
 
 This section contains technical details about the material system in the Atom Renderer.
 
+For a high level overview of the material system, see [Material System Overview](/docs/atom-guide/look-dev/materials/materials/)
+
 | Contents                        | Details |
-|--------------------------------------|---------|
-| [Material System Overview](materials/) | An overview of Atom's material system. |
+|---------------------------------|---------|
+| [Lua Material Functor API](lua-material-functor-api.md) | Lua material functors allow custom logic for processing material properties. The API is detailed here. |
```

Comment:
At the bottom of this page, add a related topics section that links to the Material System overview. Users on this page may want to access the Material System Overview page or even remember it exists.

```
## Related topics
| Related topics | Details | 
| --- | --- |
| | [Material System Overview](materials/) | An overview of the material system and related concepts. |
```
Note: If you change the file name as suggested in file `content/docs/atom-guide/look-dev/materials/materials.md`, make sure to update this too.